content/docs/user-guide/networking/_index.md

```diff
@@ -1,30 +1,62 @@
 ---
 title: Networking
 weight: 950
-description: The user guide for Open 3D Engine's networking stack.
+description: Learn about the low-level networking stack in Open 3D Engine and the Multiplayer Gem that uses it to provide multiplayer features in games and simulations.
 ---
 
-Open 3D Engine uses an abstracted interface layer, the `AzNetworking` framework, to provide access to OS-level networking interfaces. This layer provides access to features like network compression, encryption, and reliable UDP packets.
+With **Open 3D Engine (O3DE)**, you can use either of the following two networking frameworks depending on your needs:
 
-For a quick introduction to the O3DE network layer and Multiplayer Gem, watch the video below.
+* The `AzNetworking` framework, which provides a low-level networking API.
+* The **Multiplayer Gem** framework, which is built on top of `AzNetworking` and provides multiplayer networking features for games and simulations.
+
+## Networking frameworks
+
+### AzNetworking
+
+`AzNetworking` is an OS-level network interface that provides simple, fast, and efficient networking by focusing on reduced code size and complexity, low latency on packet send and receive operations, and low message processing overhead. It provides features like:
+
+* Network compression
+* Encryption
+* Serialization
+* High performance message processing
+* Reliable UDP packets
+* Wrapper classes for managing UDP and TCP sockets
+
+Refer to the [AzNetworking section](aznetworking/) topics and use the [AzNetworking API Reference](/docs/api/frameworks/aznetworking/annotated.html) if you need to work with networking classes at this low level.
+
+### Multiplayer Gem
+
+The Multiplayer Gem supports entity-based asynchronous networking in O3DE using event-driven network properties and remote procedure calls to synchronize state. The Gem includes the following features commonly needed by multiplayer games and other simulations:
+
+* Server authoritative model for anti-cheat
+* Entity replication
+* Push-based synchronization
+* Event-driven network properties
+* Reliable and unreliable remote procedure calls
+* Local prediction
+* Packet reconciliation
```

Comment:
Would consider packet reconciliation a part of transport ie how reliable UDP works but maybe theres a different  thing we are trying to talk about here.

Reply:
Nah you're right, I'll remove it. It was a nod to what's going on in the server authoritative model, as described by Karl in the Networking Overview video, but probably more of a description of the workflow rather than a separate feature.